interpreter/instructions.go

```diff
@@ -197,6 +197,21 @@ func NewJump(exprID int64, instructionCount int, cond func(EvalState) bool) *Jum
 		OnCondition:     cond}
 }
 
+// NoOp represents a no operational placeholder.
```

Comment:
Per the earlier comments, I'm not sure you need the `NoOp`. Give it a try without them and if it works, then remove this code, and if it doesn't please just rephrase the code as though short-circuiting is enabled by default, e.g.
```
inst := NewJump(...)
if !w.shortcircuit {
  inst := NewNoOp()
}
instructions = append(instructions, inst)
```

Reply:
Done. NoOp is removed